### ISSUE TITLE
Disconnection

### ISSUE THREAD

**Issue by NateDogg1232** [MarcuzD/HorizonM#14]
First, I just wanted to say OMG I CAN'T BELIEVE IT'S FINALLY BEEN DONE, SCREEN MIRRORING FOR THE o3DS

Second, I was paying attention to how the LED is and how it responds to screen touches, button presses, etc, and it seems to lose connection for a very small (and I mean very small) amount of time, then reconnect quickly following. I don't know if this could potentially cause any issues with the frames being sent or not, but it may be something worth looking into if you haven't already.

**Comment by FateForWindows** [comment]
Yeah, I noticed this too. This might help increase the framerate by a small amount.

**Comment by MarcuzD** [comment]
if by "small amount of time" you meant ~5-10seconds then that's perfectly normal, the 3DS just attempts to reconnect to the internet, and is something I can't control

if you meant the flashing when you press a button, it's purely for aestethical and debugging purposes, and it doesn't affect the rest of the program in any way. the RGB LED is just for debugging and monitoring the current state of the program
